.github/workflows/publish-to-fig-autocomplete.yml

```diff
@@ -0,0 +1,37 @@
+name: Publish to fig autocomplete
+
+on:
+  workflow_call:
+  workflow_dispatch:
+    input:
+      notUsed:
+        type: string
+        description: Run this workflow from a "release-*" branch. This input does nothing.
+        required: false
+
+jobs:
+  push-to-fig-autocomplete:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Use Node.js 16.x
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16.x
+          cache: yarn
+      - run: yarn --immutable --network-timeout 1000000
+      - name: Install Fig Oclif Plugin
+        run: ./bin/run plugins:install @fig/oclif-complete@2
+      - name: Generate Fig Spec
+        run: ./bin/run generate-fig-spec > spec.ts
+      - name: Get heroku version
+        id: cli-version
+        run: echo "version=$(./bin/run --version | sed -rn 's/^heroku\/([0-9\.]+).*$/\1/p')" >> $GITHUB_OUTPUT
+      - name: Create Fig Autocomplete PR
+        uses: withfig/push-to-fig-autocomplete-action@v1
+        with:
+          token: ${{ secrets.BOT_TOKEN }}
```

Comment:
This will need a github bot account to make PRs to the https://github.com/withfig/autocomplete repo